spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_target_vm_spec.rb

```diff
@@ -66,6 +66,32 @@
     assert_storage(storage, vm)
   end
 
+  context 'vm migration' do
+    before(:each) do
```

Comment:
I had to make a new vcr recording, I do not know how to get rid of this block. Do I understand correctly and you want me to move this context into its own spec file? just seemed to me this is the right logical place for it...